html/infrastructure/urls/resolving-urls/query-encoding/resources/css-tmpl.py

```diff
@@ -1,5 +1,5 @@
 def main(request, response):
-    encoding = request.GET['encoding']
-    tmpl = request.GET['tmpl']
-    sheet = tmpl % u'\\0000E5'
-    return [("Content-Type", "text/css; charset=%s" % encoding)], sheet.encode(encoding)
```

Comment:
It seems you removed functionality here.

Reply:
Thanks very much for spotting this! I removed encode() in a couple of places some time ago while fiddling with the codes but forgot to put them back. 

Codes have now been updated.